src/pages/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,29 @@
+import { ButtonSize, ROUTE, VARIANT } from '@common';
+import { Background, Button } from '@components';
+import { RootLayout } from '@layouts';
+import { Link } from 'react-router-dom';
+
+export const NotFound = () => {
+  return (
+    <RootLayout>
+      <Background />
+      <main className="relative z-2 w-full max-w-7xl mx-auto px-5 flex flex-col justify-center items-center h-[87vh] text-cWhite font-dmsans gap-16">
```

Comment:
```suggestion
      <main className="relative z-2 w-full max-w-7xl mx-auto px-5 flex flex-col justify-center items-center h-[87svh] text-cWhite font-dmsans gap-16">
```

Con `svh` también hacemos que se vea bien en mobile